package.json

```diff
@@ -8,7 +8,7 @@
   ],
   "scripts": {
     "build": "node scripts/build.js",
-    "test": "mocha --recursive",
+    "test": "mocha --recursive && (cd test_flow && npm install && npm test)",
```

Comment:
:/

It's not ideal since I often pass extra args to `npm test` when I'm debugging. I could come up with a workaround, though.